public/assets/tasks/style.css

```
@@ -0,0 +1,1672 @@
+.description-content li {
+    margin-bottom: 8px;
+}
+
+.description-content blockquote {
+    border-left: 4px solid var(--primary-500);
+    padding-left: 20px;
+    margin: 20px 0;
+    color: var(--gray-600);
+    font-style: italic;
+    background: var(--primary-25);
+    padding: 16px 20px;
```

Biome (2.1.2)

[error] Unexpected shorthand property padding after padding-left



(lint/suspicious/noShorthandPropertyOverrides)